src/test/resources/features/allergies.feature

```diff
@@ -0,0 +1,39 @@
+Feature: allergies feature
+
+  Background: 
+    Given User logs in, searches John doe and visits first patient dashboard
+    And User clicks on edit icon near allergies label
+
+  @run
+  Scenario: Launching allergies page
+    Then system loads patients allergies page
```

Comment:
move this into background since its a result of the former action not a scenario of its own